pmlc/conversion/gpu/gpu_to_vulkan.cc

```diff
@@ -237,10 +262,86 @@ Value ConvertGpuLaunchFuncToVulkanCalls::createEntryPointNameConstant(
                                   shaderName, LLVM::Linkage::Internal);
 }
 
+llvm::SmallSet<Operation *, 4>
+ConvertGpuLaunchFuncToVulkanCalls::getExternalDependentOperations(
```

Comment:
Just noticing that there is a lot of overlap between this function and the next function `isInternalOperation` and the analysis code in pmlc/dialect/pxa/analysis/uses.cc.  Wondering if we could / should find a more general place for this code so that it can be reused later.